app/controllers/lines_controller.rb

```diff
@@ -1,14 +1,29 @@
 # Single-serving controller for setting current line for a user.
 class LinesController < ApplicationController
   def new
-    if LINES.count == 1
-      session[:line] = LINES[0]
+    @lines = ActsAsTenant.current_tenant.lines
+    if @lines.count == 0
+      raise Exceptions::NoLinesForFundError
+    end
+
```

Comment:
I think this is the only bit of logic change in the app - throws a specific error if no line is setup.